templates/kratos.yaml.j2

```diff
@@ -1,3 +1,26 @@
+{%- set allowed_oidc_provider_configurations = [
```

Comment:
Is this required? The `kratos_external_provider` interface already validates the configuration.